src/tests/multihost/alltests/test_sssctl_analyzer.py

```diff
@@ -295,3 +304,38 @@ def test_analyze_parse_child_logs(self, multihost, backupsssdconf):
             assert re.findall(r"RID#[0-9]*] Received error code", stdout)
         else:
             pytest.fail(f"{user} sucessful to login")
+
+    def test_non_root_privileged(self, multihost, backupsssdconf, localusers):
```

Comment:
The self is not used/needed , add @staticmethod decorator to the test and remove self as a parameter.

Reply:
ok added